liboptv/include/segmentation.h

```diff
@@ -0,0 +1,47 @@
+/*  Objects related to calibration: exterior and interior parameters, glass
+    parameters. A Calibration object to rule them all and find them. 
+
+    References:
+    [1] Th. Dracos (editor). Three-Dimensional Velocity and Vorticity Measuring
+        and Image Analysis Techniques. Kluwer Academic Publishers. 1996. 
+        pp.165-168
+*/
+
+#ifndef SEGMENTATION_H
+#define SEGMENTATION_H
+
+#include "tracking_frame_buf.h"
+#include "parameters.h" 
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+
+typedef struct
+{
+  int 	pos, status;
+  short	xmin, xmax, ymin, ymax;
+  int   n, sumg;
+  double  x, y;
+  int   unr, touch[4], n_touch;	/* unified with target unr, touching ... */
+}
+peak;
+
+typedef struct
+{
+  short	       	x,y;
+}
+targpix;
```

Comment:
This internal structure can safely be replaced with a 2D array. And since it is used only internally, it can be typedeffed in the .c file.